pages/cancel/[uid].tsx

```diff
@@ -138,10 +144,10 @@ export default function Type(props) {
   );
 }
 
-export async function getServerSideProps(context) {
+export const getServerSideProps = async (context: GetServerSidePropsContext) => {
```

Comment:
Align with how [most] other files are written. Will eventually add a linting rule to enforce alignment on root-level `const x = () => {}` vs `function x() {}`.

```suggestion
export function getServerSideProps(context: GetServerSidePropsContext) {
```